Table/models/table-column.model.ts

```diff
@@ -517,8 +518,8 @@ export class TableColumn<T = IItem> implements IItemBase<T> {
     // Filtering
     this.filters = col.filters ? col.filters : []
     this.filterComponent = col.filterComponent
-    this.comparators = col.comparators
-    this.comparator = col.comparator || this.comparator
+    this.comparators = col.comparators ?? config.dataTypes.comparators[this.dataType]
+    this.comparator = col.comparator ?? config.dataTypes.defaultComparator[this.dataType] ?? this.comparator
```

Comment:
the `defaultComparator` needs to be added to the `components-config.ts` also

Reply:
im not sure in general about this... we alredy have the `COMPARATORS_BY_DATATYPE_MAP` that is used for resolving the comparators...